input/community/speakers/mark-michaelis.md

```diff
@@ -0,0 +1,40 @@
+Title: Mark Michaelis
+Location: Spokane, WA
+Email: mark@intellitect.com
+Language:
+  - English
+Topics:
+  - .NET
+  - ASP.NET 
+  - ASP.NET Web API 
+  - Architecture 
+  - Artificial Intelligence 
+  - Azure 
+  - Blazor 
+  - C# 
+  - DevOps 
+  - Machine Learning 
+  - Microsoft 365 
+  - ML.NET 
+  - Open Source 
+  - Product Management 
+  - Visual Studio 
+  - Visual Studio Code 
+  - Web Development 
+  Blog: https://intellitect.com/mark
+  Feed: http://feeds.feedburner.com/intellitect/mark
+  Twitter: https://twitter.com/markmichaelis
+  GitHub: https://github.com/markmichaelis
+  LinkedIn: https://www.linkedin.com/in/markmichaelis
+  MeetUp: https://www.meetup.com/Spokane-NET-User-Group
+  YouTube: https://www.youtube.com/channel/UCZSEfrUQnLLohBWDKRRSohw
+  Instagram: https://www.instagram.com/team_intellitect/
+  Mentor: true
```

Comment:
```suggestion
Blog: https://intellitect.com/mark
Feed: http://feeds.feedburner.com/intellitect/mark
Twitter: https://twitter.com/markmichaelis
GitHub: https://github.com/markmichaelis
LinkedIn: https://www.linkedin.com/in/markmichaelis
MeetUp: https://www.meetup.com/Spokane-NET-User-Group
YouTube: https://www.youtube.com/channel/UCZSEfrUQnLLohBWDKRRSohw
Instagram: https://www.instagram.com/team_intellitect/
Mentor: true
```

Reducing the indent on these so that they'll line up with the other fields to avoid a yaml parsing issue.